docs/pages/customers/coupa.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Coupa
+description: Delivering features 50% faster with MUI Components.
+image: '/static/branding/companies/coupa_spotlight.svg'
+date: 2022-06-08T00:00:00.000Z
+rank: '3'
+tags: ['MUI X']
+manualCard: true
+---
+
+<style>
+  #blog-responsive-image {
+    height: 230px;
+    @media (max-width: 600px) {
+      height: 167px;
+    }
+  }
+</style>
+
+# Coupa | Delivering features 50% faster with MUI Components
+
+<img
+  id="blog-responsive-image-light"
+  src="/static/branding/companies/headers/coupa-header.png"
+  alt="Coupa logo"
+  style="width: 100%; height: auto; object-fit: cover; object-position: top left; border: 0px; margin-left: 0; margin-bottom: 20px; display: block; text-align: left;"
+/>
+
+## Overview
+
+[Coupa Software](https://www.coupa.com/) is a cloud-based spend management platform that helps businesses streamline procurement, invoicing, expense management, and supplier management. They needed to modernize their frontend components to meet strict deadlines for a new mid-market product. They selected MUI to replace their outdated in-house component library, leveraging MUI's comprehensive ecosystem and advanced components.
+
+<span class="only-light-mode">
+<blockquote style="margin: 32px 0; padding: 24px 32px; background: #f5f5f7; border-left: 6px solid var(--muidocs-palette-primary-main); border-radius: 8px; font-size: 1.15rem; font-style: italic;">
+  We had very strict and aggressive deadlines, and we were able to meet them. If we had done it using our own components, it probably would have been at least six months to eight months over the deadline. So, with MUI we got 50% faster. But when you put that on a longer period, I think that number is bigger because of maintenance and shifting paradigms and business needs.
```

Comment:
```suggestion
  We had very strict and aggressive deadlines, and we were able to meet them. If we had done it using our own components, it probably would have been at least six to eight months over the deadline. So, with MUI we got 50% faster. But when you put that on a longer horizon, I think that number is bigger because of [the benefits when tackling] maintenance, and shifting paradigms and business needs.
```